services/userlog/pkg/service/service.go

```diff
@@ -90,6 +90,18 @@ func (ul *UserlogService) MemorizeEvents(ch <-chan events.Event) {
 		switch e := event.Event.(type) {
 		default:
 			err = errors.New("unhandled event")
+		// file related
+		case events.PostprocessingStepFinished:
+			if e.FinishedStep != events.PPStepAntivirus {
+				continue
+			}
+			result := e.Result.(events.VirusscanResult)
+			if !result.Infected {
+				continue
+			}
+
+			// TODO: should space mangers also be informed?
```

Comment:
who do you need to discuss and decide that?

Reply:
Just another opinion than my own